### ISSUE TITLE
When indicated to be present by surface samples and dowsing rod, gold isn't always present.

### ISSUE THREAD

**Issue by EhGneissGuy** [Darkosto/SevTech-Ages#1992]
This was an issue in a previous version I believe (3.03 I think) but it was marked as closed. However, this issue is still present.

## Issue / Bug
Dowsing rod and surface samples indicate presence of gold in chunk. Even after mining out entirety of the chunk and into the walls of surrounding chunks, gold deposit is not actually present. Sometimes gold does properly generate without issue, but it just seems hit or miss.

## Expected Behavior
Ore to be present in the chunk it's indicated to be in.

## Steps to Reproduce (for bugs)
1. Generate world in 3.06 with suggested settings.
2. Reach age where gold is detectable.
3. Find chunk that indicates deposit.
4. Dig normally, find no gold.
4. After digging out entire chunk, ore still cannot be found.

## Client Information
* Modpack Version: 3.06
* Java Version: 8 - 171
* Launcher Used: ATLauncher
* Memory Allocated: 4GB
* Server/LAN/Single Player: Server
* Resourcepack Enabled?: No
* Optifine Installed?: Yes

## Server Information

Going to be straight with you - I honestly don't know. It's a CubedHost package, 4GB. Runs like a dream, though.

* Java Version: 
* Operating System:
* Hoster/Hosting Soultion: CubedHost, 4GB
* Sponge (Non-Vanilla Forge) Server?:

* Additional Content Installed?: No

**Comment by pepi-tsankov** [comment]
Hello. I had a similar "issue". Since gold spawns between layers 8 and 30, it can more often than not be invisible to the dowsing rod, since it detects in range of 48. My base was at a height of 70, there were some gold samples nearby. I decided to go down next to one of them and just use the dowsing rod in a bunch of spaces. I found it pretty easily. I believe this is not a bug, just gold being in itself hard to find because of its low y value of spawning.

**Comment by EhGneissGuy** [comment]
Pepsi-tsankov, thanks for sharing your experience. However, after going into creative and knocking out every block in the chunk, there is no gold whatsoever despite being indicated by the dowsing rod and surface samples. What I'm saying ia the dowsing rod and surface samples both indicate a vein, but when all blocks in the chunk/surrounding chunks are mined out, it simply isn't there.

**Comment by EhGneissGuy** [comment]
Dutchs, I'll try that out asap. Will reply if it works.

**Comment by EhGneissGuy** [comment]
Dutchs, the gold is not below bedrock.

**Comment by SphynxLoL** [comment]
Try using Scannable's "Scanner" if you're at that advancement. I used to have a lot of trouble finding the deep ores with the dowsing rod until I got the scanner. I've found every node since.

To add, sometimes there may only be 1-2 blocks of the ore inside the chunk that is showing samples on top. From there it can completely extend into a neighboring chunk. That is why the scanner is a way better method of locating ore viens.
